src/po/Make_mvc.mak

```diff
@@ -28,9 +24,9 @@ VIM = ..\vim
 # Correct the following line for the directory where gettext et al is installed
 GETTEXT_PATH = D:\Programs\GetText\bin
```

Comment:
@k-takata  , would you be able to add the following comment above this line?
"`# Please do not put the path in quotes.`"